### ISSUE TITLE
chaos-dashboard: nowhere to setup "direction" during creating NetworkChaos Experiment

### ISSUE THREAD

**Issue by STRRL** [chaos-mesh/chaos-mesh#1235]
## Bug Report

**What version of Kubernetes are you using?**

kuberentes v1.9.4
minikube version: v1.15.1
macos 11.0.1

chaos-mesh release v1.0.2

**What did you do?**

- install chaos-mesh via `curl -sSL https://mirrors.chaos-mesh.org/v1.0.2/install.sh | bash`
- `k apply -f https://raw.githubusercontent.com/paulbouwer/hello-kubernetes/master/yaml/hello-kubernetes.yaml`
- try to create a Network Chaos between one pod with another pod, **with direction: to**

**What did you expect to see?**

**What did you see instead?**
Nowhere to setup direction:
![image](https://user-images.githubusercontent.com/20221408/100706427-f707ac00-33e3-11eb-810f-87413282ca1d.png)

**Comment by STRRL** [comment]
Master branch does not have this issue.

How about cherry-pick the new UI of "creating chaos experiments" to release-1.0?

**Comment by g1eny0ung** [comment]
> Master branch does not have this issue.
> 
> How about cherry-pick the new UI of "creating chaos experiments" to release-1.0?

It's hard to migrate the latest changes into the release `1.0.x`. 😢 I am looking forward to the release of `v1.1`.

**Comment by wingyiu** [comment]
should also update NetworkChaos example, add direction field

**Comment by fewdan** [comment]
> should also update NetworkChaos example, add direction field
> 
> ```
> status:
>   experiment:
>     phase: Failed
>   failedMessage: 'unknown direction '
>   phase: ""
>   scheduler: {}
> ```

@wingyiu 
Hello, can you tell me which example you use? It would be better if the steps for replication could be provided.
Thank you!

**Comment by victoriacheng15** [comment]
👋 @g1eny0ung @STRRL 

Checking in on this during triage. I noticed that the `direction` selector (`to`, `from`, `both`) is now supported in the UI across all NetworkChaos actions, and the API/CRD defaults to `to` when omitted.

I think this can be closed, unless there is an edge case or detail I might have missed? 👀

**Comment by g1eny0ung** [comment]
@victoriacheng15 I agree with your conclusion. It seems this issue should be closed, we didn't specifically fix it through a PR before. I'll close it now.
